### DISCUSSION TITLE
Add new class Demon Hunter with Legion 7.3.5

### DISCUSSION THREAD

**Discussion by jimteck** [trickerer/Trinity-Bots#419] [Classes]
I suggest adding a new class demon hunter (take the model or the ilidarii near the black temple) take class abilities similar to the demon hunter from 7.3.5 or already suggest your options.

**Comment by trickerer** [comment]
Sorry, I didn't play anything beyound 4.x.x so I don't know anything of 7.3.5 Demon Hunters and their abilities. Reading the lore of any new classes introduced since classic almost makes me retch.

The best thing I can do is implement W3 Demon Hunter, but unfortunately last time I was thinking about how to do that I ran into several problems:
1. No proper / fitting Night Elf model for the class, it has to be different from common one I think; there is one from Nagrand guy but not sure about that one. W3 DH attack animation was way too specific.
2. No reusable abilities for any of DH spells. As far as I know there is no visual anywhere similar to W3 Mana Burn, Metamorphosis also was impossible to make at that time since spells could not be reused as bot spells - only borrowed as a whole.
3. Metamorphosis form attack animation was way too fancy for WoW developers it seems and demon form model does not have that at all.
